packaging/greenboot/functions.sh

```diff
@@ -294,3 +294,94 @@ function log_script_exit() {
         echo "FINISHED"
     fi
 }
+
+# lvmsDriverShouldExist shallowly wraps some python yaml parsing to determine if the LVMS-Operator is expected to be
+# deployed.
+# args:
+#   $1 microshift config file path
+# return:
+#   0:  "true", expect the driver to be running
+#   1:  "false", do not expect the driver to be running
+function lvmsDriverShouldExist() {
+    # The python script checks the config's .storage.driver value. Parsing errors are ignored, these will be caught
+    # by MicroShift during startup, before this greenboot script is run. The script favors verbosity over simplified
+    # code.
+    # args:
+    #   None
+    # return:
+    #   None
+    # exit code:
+    #   0: The lvms components should be checked for. This case occurs for any one of:
+    #       - .storage.driver == one of ["", "lvms"]
+    #       - .storage.driver is unset (indicates default config)
+    #       - .storage is unset (indicates default config)
+    #       - the file cannot be opened (indicates default config)
+    #   1: The lvms components should not be checked for. This case occurs for:
+    #       - .storage.driver == "none"
+    if ! python -c \
```

Comment:
Can we get both ifs to have the same "logic"? Currently this one is "negative", the other one is "positive"

Reply:
I did the opposite and dropped the exit code inversion :D The return values also reflect the change. If script succeeds, return 0, else return 1.